### ISSUE TITLE
Hello there, can I see the train set?

### ISSUE THREAD

**Issue by yiran7324** [deeppomf/DeepCreamPy#64]
Or how can I collect the original image to train the neural network?

**Comment by deeppomf** [comment]
See https://www.gwern.net/Danbooru2017 for a starting place for the data.
I don't want to disclose the sources of the data because if enough people try scraping the images, scraping will get blocked. Tragedy of the commons, so to speak.
At the same time, I can't share the data because it's all copyrighted content.

**Comment by Kadantte** [comment]
@deeppomf About create our own Data are u gonna share anything like what u was sharing on DeepMindBreak ?! Just for update the Model ...

**Comment by deeppomf** [comment]
@Kadantte What do you want me to share from DeepMindBreak? I could post DeepMindBreak in a separate repo.

**Comment by Kadantte** [comment]
@deeppomf I've the latest version of DMB, and what i need u to share is training option for DCP like u do with DMB.

**Comment by Yukariin** [comment]
@deeppomf do you use same mask generation procedure as in original PConv project?

**Comment by deeppomf** [comment]
@Yukariin 
My mask generation procedure is different from https://github.com/MathiasGruber/PConv-Keras
The mask generation procedure from the original paper is better.

@Kadantte 
I don't intend to release the training code.
I want all work to this project to be AGPL.
If I release the training code under AGPL, that doesn't mean neural network models generated from the training code are AGPL. It's like how works made with Blender, which a GPL 3D animation software, aren't GPL even though Blender is GPL.
Plus, my training code is tailored to the dataset I have and wouldn't make sense with other datasets.

If there's a way to enforce AGPL on the generated neural networks, then the training code will be released.

**Comment by Kadantte** [comment]
@deeppomf Yeah that make since you're right, anywway i hope if there is a way to keep it on AGPL.

**Comment by marcussacana** [comment]
@deeppomf  I don't understand much about license, but by what I can see you don't want stop use the AGPL in the DeepCreamPy, and if you share the training tool you can't use AGPL, rigth?
So, you can't just make a new repo only with the training tool with the rigth license?

Well If i'm wrong you can do a 'bigger' optional model? actually have one model of 125mb, I hear you don't want make he bigger, but if is this the case you can't put as a 'optional model'?

**Comment by Kadantte** [comment]
@marcussacana Yeah if he can to provide us with training tool it' will be nice since a lot of ppl want to help with it.

**Comment by Yukariin** [comment]
@Kadantte well, you can get training code from [PConv](https://github.com/MathiasGruber/PConv-Keras/blob/master/Step4%20-%20Imagenet%20Training.ipynb), just edit (or not) data generator with your own mask generator.

**Comment by Kadantte** [comment]
@Yukariin Thank's I'll look into it .
